### PR TITLE
Add python-libsbml-experimental

### PR DESCRIPTION
**Pull request by Midnighter**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/python-libsbml-experimental/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0d532219867a0bef09785f100dc272f96e516a2f512cb9b3219197bef31465af
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
```
Comment (scopatz) [inline recipes/python-libsbml-experimental/meta.yaml]: Should use `skip: true  # [py < 35]`

File: recipes/python-libsbml-experimental/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python >=3.5
```
Comment (scopatz) [inline recipes/python-libsbml-experimental/meta.yaml]: This and the line below should just be `- python`

File: recipes/python-libsbml-experimental/meta.yaml (hunk 1 above)
Comment (Midnighter) [reply recipes/python-libsbml-experimental/meta.yaml]: Thank you for taking a look. :smiley:

**Comment by Midnighter** [comment]
Any idea about the linker not being found on Windows @scopatz ?

**Comment by scopatz** [comment]
I am not a windows expert, sorry.  ping @conda-forge/help-c-cpp @conda-forge/staged-recipes

**Comment by Midnighter** [comment]
I found out that the linker is actually not missing but that the command is too long for the Windows shell. I am currently working for a change in the python-ibsbml build system. They are making a switch to a cmake-based build which should avoid this problem. I will update the PR when the upstream code is ready.

**Pull request closed without merging**